apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -219,6 +220,10 @@ export class BulkScriptAction extends BaseScriptAction {
           'but not any projects that directly or indirectly depend on the changed package.'
       });
     }
+    this._ignoreHooksParameter = this.defineFlagParameter({
+      parameterLongName: '--ignore-hooks',
+      description: `Overrides execution of event hooks. Meant for use on CI machines.`
```

Comment:
```suggestion
      description: `Skips invocation of the "eventHooks" scripts that are defined in rush.json. This maybe useful for automated jobs.`
```

Reply:
Applied the first half. I prefer the current version of the second half (no statement of where you want it, just to be careful using it).